demos/sev-demo/README.md

```diff
@@ -0,0 +1,136 @@
+
+## OVMF
+
+You will need to use OVMF. Furthemore, you will need to use the AMDSEV build of OVMF. An appropriate firmware binary is available in this directory. You can also build from source. To do so, follow the standard OVMF build procedure, but use `edk2/OvmfPkg/AmdSev/AmdSevX64.dsc` rather than the OVMF DSC file. The AMDSEV build currently expects to use Grub for reasons unrelated to Kata. To bypass this, you should comment out line 28 of `edk2/OvmfPkg/AmdSev/AmdSevX64.dsc`. If the build process complains about a missing `grub.efi` file, you can create a dummy file with that name and try again. This dependency may be adjusted in the future and we are hoping to provide a prebuilt version of the appropriate firmware.
+
+## Rootfs
+ 
+Create the rootfs as you normally would for CCv0, but be sure to specify set `AA_KBC` to `offline_sev_kbc` so that the AA is built with the correct key broker client. 
+
+```
+sudo make USE_DOCKER=yes DISTRO=ubuntu AGENT_INIT=yes DOCKER_RUNTIME=runc AA_KBC=offline_sev_kbc SKOPEO_UMOCI=yes rootfs
+```
+
+You will need to add the agent configuration file to the initrd. Again, this is a typical CCv0 requirement, but you will need to specify parameters for you KBC. Specifically be sure to add this parameter.
+```
+aa_kbc_params = "offline_sev_kbc::null"
+```
+
+The `offline_sev_kbc` uses the `efi_secret` kernel module to expose injected secrets to the filesystem. You'll need to add the kernel module to the initrd. You can do this by setting the `KERNEL_MODULES_DIR` when you build your rootfs or you can copy the kernel module directly into the rootfs that is created. Either way, your rootfs should look something like this. 
+```
+$ tree ubuntu_rootfs/lib/modules
+lib/modules
+└── 5.15.0-rc5+
+    ├── kernel
+    │   └── drivers
+    │       └── virt
+    │           └── coco
+    │               └── efi_secret
+    │                   └── efi_secret.ko
+    ├── modules.alias
+    ├── modules.alias.bin
+    ├── modules.builtin
+    ├── modules.builtin.alias.bin
+    ├── modules.builtin.bin
+    ├── modules.builtin.modinfo
+    ├── modules.dep
+    ├── modules.dep.bin
+    ├── modules.devname
+    ├── modules.order
+    ├── modules.softdep
+    ├── modules.symbols
+    └── modules.symbols.bin
+```
+
+Make sure that the module matches your kernel version. Speaking of which, `efi_secret` also requires some minor kernel patches. These patches and the module are not currently upstream. You will need to find Dov Murik's EFI secret patches on the kernel mailing list and build a kernel with them. This should also give you the correct version of the `efi_secret` module.  
+
```

Comment:
I think we'd at least need a URL to Dov's patches, even if it's a mailing list link.

Reply:
> the correct version

What is the correct version? Or do you just mean "a version" that will actually work for the demo?